src/main/java/de/th/koeln/archilab/fae/faeteam3service/ausnahmesituation/AusnahmesituationController.java

```diff
@@ -27,11 +31,25 @@ public Ausnahmesituation createAusnahmesituation(@Valid @RequestBody Ausnahmesit
         ausnahmesituation.addNachricht(neueNachricht);
 
         nachrichtRepository.save(neueNachricht);
-
         log.info("Ausnahmesituation erstellt: " + ausnahmesituation.toString());
+
+        sendMailToKontaktperson(neueNachricht);
+
         return ausnahmesituation;
     }
 
+    private void sendMailToKontaktperson(Nachricht nachricht) {
```

Comment:
Ich denke, dass wir das in Zukunft in einen eigenen Service auslagern können?

Und (in Zukunft) sollte als Parameter denke ich nicht nur die Nachricht, sondern auch die Kontaktperson und die Erkrankte Person übergeben werden, damit wir (wenn wir die Daten dann auch haben) die uns übergebenen Daten verwenden können.

Reply:
Da hast du vollkommen Recht, habe es gerade in die `EmailServiceImpl` Klasse gepackt.
Und bezüglich der Parameter habe ich uns mal ein TODO dran gesetzt, das wir abarbeiten können sobald uns die nötigen Daten zur Verfügung stehen.